src/compas_viewer/components/objectsetting.py

```diff
@@ -86,11 +51,23 @@ class ObjectSetting(QWidget):
 
     update_requested = Signal()
 
-    def __init__(self, viewer: "Viewer"):
+    def __init__(self, viewer: "Viewer", items: list[dict]):
         super().__init__()
         self.viewer = viewer
-        self.layout = QVBoxLayout(self)
-        self.spin_boxes = {}
+        self.items = items
+        self.setFixedHeight(200)
```

Comment:
We should not set a fix height, the form should be allowed to adjusted vertically like this:
(also can you make the items vertically align to the top?)
<img width="269" alt="image" src="https://github.com/user-attachments/assets/c4d5ca05-87b0-4144-a176-06fd1a3c290c">